src/js/slider/slider.js

```diff
@@ -180,7 +180,7 @@ class Slider extends Component {
     this.on(doc, 'touchmove', this.handleMouseMove_);
     this.on(doc, 'touchend', this.handleMouseUp_);
 
-    this.handleMouseMove(event);
+    this.handleMouseMove(event, true);
```

Comment:
If it's something we may consider for the future, it may be worth adding a TODO for splitting the seeking logic out of the mouse move handler and calling it separately from here.

Reply:
Created https://github.com/videojs/video.js/issues/7531 to follow up on this.